### PR TITLE
SONARPY-2399 update rules tags

### PR DESCRIPTION
**Pull request by maksim-grebeniuk-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S1129/python/metadata.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "constantCost": "2min"
   },
   "tags": [
-
+    "convention"
```
Comment (thomas-serre-sonarsource) [inline rules/S1129/python/metadata.json:16]: S1129 is not in our google sheet. Is it a type?

File: rules/S1143/python/metadata.json [hunk 2]
```diff
@@ -1,3 +1,8 @@
 {
-  "title": "Break, continue and return statements should not occur in \"finally\" blocks"
+  "title": "Break, continue and return statements should not occur in \"finally\" blocks",
+  "tags": [    
+    "cwe",
+    "error-handling",
+    "pitfall"
```
Comment (thomas-serre-sonarsource) [inline rules/S1143/python/metadata.json:6]: S1143 should already have `cwe` and `error-handling`. Maybe you should update the top level meta data json?
I think there are several rule update where I have this comment but I'll comment only this one.

File: rules/S1244/python/metadata.json [hunk 3]
```diff
@@ -13,7 +13,10 @@
     "constantCost": "5min"
   },
   "tags": [
-
+    "suspicious",
+    "numpy",
+    "pytourch",
```
Comment (thomas-serre-sonarsource) [inline rules/S1244/python/metadata.json:18]: pytourch :sweat_smile:  -> pytorch

File: rules/S3699/python/metadata.json [hunk 4]
```diff
@@ -1,3 +1,5 @@
 {
-  
+  "tags": [
+    "pithon3"
```
Comment (thomas-serre-sonarsource) [inline rules/S3699/python/metadata.json:3]:
```suggestion
    "python3"
```

File: rules/S5719/python/metadata.json [hunk 5]
```diff
@@ -9,10 +9,13 @@
   },
   "status": "ready",
   "remediation": {
-    "func": "Constant\/Issue",
```
Comment (thomas-serre-sonarsource) [inline rules/S5719/python/metadata.json:12]: mistake to remove this `\`?

File: rules/S6794/python/metadata.json [hunk 6]
```diff
@@ -1,12 +1,13 @@
 {
-"title": "Type aliases should be declared with a \"type\" statement",
+  "title": "Type aliases should be declared with a \"type\" statement",
```
Comment (thomas-serre-sonarsource) [inline rules/S6794/python/metadata.json:2]:
```suggestion
"title": "Type aliases should be declared with a \"type\" statement",
```

**Review by thomas-serre-sonarsource — CHANGES_REQUESTED** [review]
LGTM! It must have been a cumbersome PR to create :-) 
I put it back in progress, but there are only minor changes.

File: rules/S1129/python/metadata.json (hunk 1 above)
Comment (maksim-grebeniuk-sonarsource) [reply rules/S1129/python/metadata.json:16]: in our document it is mentioned with its legacy key: `LongIntegerWithLowercaseSuffixUsage`

File: rules/S1143/python/metadata.json (hunk 2 above)
Comment (maksim-grebeniuk-sonarsource) [reply rules/S1143/python/metadata.json:6]: discussed

File: rules/S6794/python/metadata.json (hunk 6 above)
Comment (maksim-grebeniuk-sonarsource) [reply rules/S6794/python/metadata.json:2]: discussed

File: rules/S1129/python/metadata.json (hunk 1 above)
Comment (thomas-serre-sonarsource) [reply rules/S1129/python/metadata.json:16]: clear, Thanks!

File: rules/S1143/python/metadata.json (hunk 2 above)
Comment (thomas-serre-sonarsource) [reply rules/S1143/python/metadata.json:6]: :+1

**Review by thomas-serre-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**